ui/admin/tests/acceptance/users/create-test.js

```diff
@@ -130,15 +130,18 @@ module('Acceptance | users | create', function (hooks) {
       );
     });
 
-    await click(`[href="${urls.newUser}"]`);
-    await fillIn('[name="description"]', 'test');
-    await click('[type="submit"]');
+    await click(commonSelectors.HREF(urls.newUser));
```

Comment:
None blocking comment: I know the code was there and a bit out of the scope of the selectors refactor. That said, this test is mocking a request where marks the field name as required, but in reality, for users name is not a required field. That means we are testing a none realistic case, but we need to test the failed API response. In these cases were the domain has no required fields, Cameron recently posted this example.
Let me know your thoughts  😉 

I know this is something out of the scope of this PR where we are refactoring test selectors. I am fine if you want to fix this on a separate PR .